src/index.js

```diff
@@ -15,10 +15,45 @@ const menu = {
 
 const table = ['Mesa 1', 'Mesa 2', 'Mesa 3', 'Mesa 4', 'Mesa 5'];
 
+function randomTime() {
+  var min = 1, max = 8;
+  var rand = Math.floor(Math.random() * (max - min + 1) + min);
+  return rand * 1000;
+}
+
 const waiter = () => {
-  orders(6000, menu.hamburger, table[3])
+  orders(randomTime(), `${menu.hamburger}`, table[3])
```

Comment:
No es necesario este cambio. 

```suggestion
  orders(randomTime(), menu.hamburger, table[3])
```